docs/source/download.rst

```diff
@@ -9,45 +9,67 @@ Thetis requires installation of `Firedrake
 principle other Linux and Linux-like systems) and must be run from
 within the Firedrake virtual environment.
 
-Install Firedrake and Thetis
------------------------------
+Installing Firedrake
+---------------------
 
-You can install both Firedrake and Thetis by running::
+You can install Firedrake by following the download documentation on the
+`Firedrake website <http://firedrakeproject.org/install.html>`_.
 
-    curl -O https://raw.githubusercontent.com/firedrakeproject/firedrake/master/scripts/firedrake-install
-    python3 firedrake-install --install thetis
+.. note::
 
-See  `Firedrake website <http://firedrakeproject.org/download.html>`_ for more
-information on the installation procedure. Note that the install proceduce may
-take up to one hour depending on your system.
+   **Installing PETSc and Firedrake can take up to 30 minutes depending on the system.**
 
-The Thetis source will be installed in the ``src/thetis`` subdirectory
-of your Firedrake install.
-In order to use Firedrake and Thetis you need to activate the Firedrake
+After installation, in order to use Firedrake and install Thetis you need to activate the Firedrake
 virtualenv::
 
-    source <your-firedrake-install-dir>/firedrake/bin/activate
+   source <your-firedrake-venv-dir>/bin/activate
 
-Using this install method you should
-**not** add Thetis to your ``PYTHONPATH``. Instead, Thetis will
-automatically be available to import whenever your Firedrake
-virtualenv is active.
+.. warning::
 
-If you have already installed Firedrake
----------------------------------------
+   **Please check that Firedrake is working before trying to install Thetis.**
```

Comment:
Instead of this I would probably say something above like: "You should check that the Firedrake install has been successful by running the command `firedrake-check`."